docs/management/connectors/action-types/openai.asciidoc

```diff
@@ -21,6 +21,18 @@ You can create connectors in *{stack-manage-app} > {connectors-ui}*.  For exampl
 image::management/connectors/images/gen-ai-connector.png[OpenAI connector]
 // NOTE: This is an autogenerated screenshot. Do not edit it directly.
 
+[IMPORTANT]
+====
+Elastic provides no official support for connecting to the Azure OpenAI service through a proxy.
+However if you must use a proxy,
+ensure that the proxy supports streaming and is SSE-compatible,
+or the connector will timeout when it attempts to communicate with the Azure OpenAI API,
+and the response will not be parsed.
```

Comment:
@dgieselaar Does this address your concern?

Reply:
The risk of a timeout is one of the reasons why we use streaming, and it's not guaranteed to timeout. The main issue is that we will not parse a non-streamed response - I think we should focus on that, and not the timeout. Docs are here: https://platform.openai.com/docs/api-reference/chat/streaming.

Reply:
Got it. Thanks for the clarification!